apps/prairielearn/src/pages/instructorCourseAdminInstances/instructorCourseAdminInstances.html.ts

```diff
@@ -0,0 +1,172 @@
+import { html } from '@prairielearn/html';
+import { renderEjs } from '@prairielearn/html-ejs';
+
+export function InstructorCourseAdminInstances({ resLocals }: { resLocals: Record<string, any> }) {
+  return html`
+    <!doctype html>
+    <html lang="en">
+      <head>
+        ${renderEjs(__filename, "<%- include('../partials/head'); %>", {
+          ...resLocals,
+          pageTitle: 'Course Instances',
+        })}
+      </head>
+      <body>
+        <script>
+          $(function () {
+            $('#earliest-access-date [data-toggle="popover"]').popover({
+              sanitize: false,
+            });
+            $('#latest-access-date [data-toggle="popover"]').popover({
+              sanitize: false,
+            });
+          });
+        </script>
+        ${renderEjs(__filename, "<%- include('../partials/navbar'); %>", resLocals)}
+        <main id="content" class="container-fluid">
+          ${renderEjs(
+            __filename,
+            "<%- include('../partials/courseSyncErrorsAndWarnings'); %>",
+            resLocals,
+          )}
+          <div class="card mb-4">
+            <div class="card-header bg-primary">
+              <div class="row align-items-center justify-content-between">
+                <div class="col-auto">
+                  <span class="text-white">Course instances</span>
+                </div>
+                ${resLocals.authz_data.has_course_permission_edit &&
+                !resLocals.course.example_course &&
+                !resLocals.needToSync
+                  ? html`
+                      <div class="col-auto">
+                        <form name="add-course-instance-form" method="POST">
+                          <input
+                            type="hidden"
+                            name="__csrf_token"
+                            value="${resLocals.__csrf_token}"
+                          />
+                          <button
+                            name="__action"
+                            value="add_course_instance"
+                            class="btn btn-sm btn-light"
+                          >
+                            <i class="fa fa-plus" aria-hidden="true"></i>
+                            <span class="d-none d-sm-inline">Add course instance</span>
+                          </button>
+                        </form>
+                      </div>
+                    `
+                  : ''}
+              </div>
+            </div>
+
+            <div class="table-responsive">
+              <table class="table table-sm table-hover table-striped">
+                <thead>
+                  <tr>
+                    <th>Long Name</th>
+                    <th>CIID</th>
+                    <th id="earliest-access-date">
+                      Earliest Access Date
+                      <a
+                        role="button"
```

Comment:
Can we replace `<a role="button" tabindex="0" ...>` with an actual `<button>`?

Reply:
Replaced `<a role="button" tabindex="0" ...>` with a `<button>` here and below in [86e4395](https://github.com/PrairieLearn/PrairieLearn/pull/9812/commits/86e4395ef9e0858a540dadab76ffab2835b365d8).